### ISSUE TITLE
Image control doesn't properly render SVG

### ISSUE THREAD

**Issue by MasonBergstrom** [microsoft/react-native-windows#9441]
### Problem Description

There seems to be an issue with the way the Image control is rendering SVG images. The images will render, but they don't look as smooth and crisp as an SVG should be. 

I discovered that if I add these two lines to [ReactImage.cpp just before setting the ImageSource](https://github.com/microsoft/react-native-windows/blob/main/vnext/Microsoft.ReactNative/Views/Image/ReactImage.cpp#:~:text=imageBrush.ImageSource(svgImageSource)%3B): 
```c++
svgImageSource.RasterizePixelHeight(strong_this->ActualHeight());
svgImageSource.RasterizePixelWidth(strong_this->ActualWidth());
```
that the images now render correctly, that's how I got the expected and actual images. Sadly, we don't have a way to set RasterizePixel values from the RN code, so there's no way to get this to always apply. In a matching XAML implementation we don't need RasterizePixel values for the SVGs to work; We've tried to get all the properties to match identically, so we're not sure why the RNW version is showing these pixelated images.

### Steps To Reproduce

1. Render SVG using RN Image control
    1. Base64 string in this case
1. See pixelated SVG image  

### Expected Results

|Expected|Actual|
|----------|-------|
|![image](https://user-images.githubusercontent.com/13530957/151637424-e7de055d-8a00-4f8e-bfd8-a348ed254cc4.png)|![image](https://user-images.githubusercontent.com/13530957/151637527-6ea97fe9-8aeb-4221-8c27-320c2a141c0f.png)|

### CLI version

6.3.1

### Environment

```markdown
System:
    OS: Windows 10 10.0.19044
    CPU: (12) x64 Intel(R) Xeon(R) W-2235 CPU @ 3.80GHz
    Memory: 29.48 GB / 63.73 GB
  Binaries:
    Node: 16.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.11 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 7.21.0 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowAllTrustedApp: Enabled
      AllowDevelopmentWithoutDevLicense: Enabled
      AllowAllTrustedApps: Enabled
      Versions: 10.0.17763.0, 10.0.18362.0, 10.0.19041.0, 10.0.20348.0, 10.0.22000.0
  IDEs:
    Android Studio: Not Found
    Visual Studio: 16.11.32002.261 (Visual Studio Enterprise 2019), 17.0.32014.148 (Visual Studio Enterprise 2022)
  Languages:
    Java: 11.0.12
  npmPackages:
    @react-native-community/cli: Not Found
    react: Not Found
    react-native: Not Found
    react-native-windows: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

### Target Platform Version

10.0.19041

### Target Device(s)

Desktop

### Visual Studio Version

Visual Studio 2019

### Build Configuration

Release

### Snack, code example, screenshot, or link to a repository

Minimal Repro:
```jsx
<Grid height={24} width={24}>
  <Image
    source={{
      uri: "data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9Ii0xMS41IC0xMC4yMzE3NCAyMyAyMC40NjM0OC" +
        "I+CiAgPHRpdGxlPlJlYWN0IExvZ288L3RpdGxlPgogIDxjaXJjbGUgY3g9IjAiIGN5PSIwIiByPSIyLjA1IiBmaWxsPSIjNjFkYWZiIi8+CiAgPGcgc3Ryb2tlPSIjNjFkY" +
        "WZiIiBzdHJva2Utd2lkdGg9IjEiIGZpbGw9Im5vbmUiPgogICAgPGVsbGlwc2Ugcng9IjExIiByeT0iNC4yIi8+CiAgICA8ZWxsaXBzZSByeD0iMTEiIHJ5PSI0LjIiIHRy" +
        "YW5zZm9ybT0icm90YXRlKDYwKSIvPgogICAgPGVsbGlwc2Ugcng9IjExIiByeT0iNC4yIiB0cmFuc2Zvcm09InJvdGF0ZSgxMjApIi8+CiAgPC9nPgo8L3N2Zz4K"
    }}
  />
</Grid>
```

I'm just using the React logo for this minimal repro, if you want access to the actual SVGs being used send me a PM.

**Comment by jonthysell** [comment]
This might be fixable with a different resize mode.

**Comment by chrisglein** [comment]
Workaround is to use fixed PNG assets in the meantime.
@marlenecota I heard that George had been investigating some blurry SVG issues. You should check with XAML Rendering to see if there's a bug in the platform here with some configurations.

**Comment by jonthysell** [comment]
Is this still something that makes sense for 0.69 or should it be moved to 0.70? Or maybe the Backlog?

**Comment by jonthysell** [comment]
Partner requests traction on this, preferably to backport a fix to 0.69.

**Comment by ChrisGuzak** [comment]
I learned this is related to this issue in the OS ADO http://task.ms/34544877
